### ISSUE TITLE
VB EnC rename tests are failing in CI

### ISSUE THREAD

**Issue by CyrusNajmabadi** [dotnet/roslyn#62844]
The following tests are failing:

```
Test	Duration
Microsoft.CodeAnalysis.VisualBasic.EditAndContinue.UnitTests.TopLevelEditingTests.PropertyRename1 Failed	4 ms
Project: Microsoft.CodeAnalysis.VisualBasic.EditorFeatures.UnitTests Failed (6)	2.1 sec
Namespace: Microsoft.CodeAnalysis.VisualBasic.EditAndContinue.UnitTests Failed (6)	2.1 sec
Class: TopLevelEditingTests Failed (6)	2.1 sec
Microsoft.CodeAnalysis.VisualBasic.EditAndContinue.UnitTests.TopLevelEditingTests.Event_Rename Failed	16 ms
Microsoft.CodeAnalysis.VisualBasic.EditAndContinue.UnitTests.TopLevelEditingTests.EventInsert_IntoLayoutClass_Sequential Failed	266 ms
Microsoft.CodeAnalysis.VisualBasic.EditAndContinue.UnitTests.TopLevelEditingTests.Method_Rename Failed	7 ms
Microsoft.CodeAnalysis.VisualBasic.EditAndContinue.UnitTests.TopLevelEditingTests.PartialMember_RenameInsertDelete Failed	52 ms
Microsoft.CodeAnalysis.VisualBasic.EditAndContinue.UnitTests.TopLevelEditingTests.PropertyRename2 Failed	1.8 sec
```

**Comment by CyrusNajmabadi** [comment]
Disabled in https://github.com/dotnet/roslyn/pull/62833

**Comment by Youssef1313** [comment]
@davidwengier fwiw, I ran the test on the branch from the PR (got it via [`gh pr checkout 62364`](https://cli.github.com/manual/gh_pr_checkout)) and it still failed locally. It's very strange this passed CI in first place.

**Comment by CyrusNajmabadi** [comment]
@RikkiGibson  @allisonchou  ^  Very concerning.  We may have  CI hole here.

**Comment by RikkiGibson** [comment]
It would be useful to know if the test failures are observed when running `.\Build.cmd -test`, or if they are only observed in e.g. Test Explorer or `dotnet test`.

**Comment by davidwengier** [comment]
Sorry! I blindly trust GitHub to not let me merge if things go bad.

Looks like my last commit/push didn't kick off a build for some reason.

**Comment by RikkiGibson** [comment]
Fixed by #62843
